src/conv/k-g/region_list.cpp

```diff
@@ -49,16 +49,53 @@ Bot& RegionList::addRegion
 }
 
 
+void RegionList::setAttributes
+(
+    const std::string&                        name,
+    const std::map<std::string, std::string>& attributes
+) {
+    m_list[name].attributes = attributes;
+}
+
+static void  writeAttributes
+(
+    rt_wdb* wdbp,
+    const char* name,
+    const std::map<std::string, std::string>& attributes
+) {
+
+    struct rt_db_internal     region_internal;
+    struct directory*         dp;
+    struct db_i*              dbip;
+    bu_attribute_value_set*   avs;
+
+    dbip = wdbp->dbip;
+    dp = db_lookup(dbip, name, 0);
+
+    rt_db_get_internal(&region_internal, dp, dbip, bn_mat_identity, &rt_uniresource);

```

Comment:
do think it's neccessary to check the Id returned by rt_db_get_internal()==ID_COMBINATION ?